gpo-examples/update_status.py

```diff
@@ -0,0 +1,27 @@
+# Use dev endpoint to update sample status
+
+from obtain_token import obtain_session
+
+# Payload to fill in
+payload = {
```

Comment:
I think we need a bit more of an example here.  Maybe a few payloads showing how to move to Processing, or lab_data_complete.

Also, we should have one to generate a snapshot